api/data_explorer/controllers/facets_controller.py

```diff
@@ -102,65 +112,3 @@ def facets_get(filter=None, extraFacets=None):  # noqa: E501
 
     return FacetsResponse(
         facets=facets, count=es_response._faceted_search.count())
-
-
-def deserialize(filter_arr, es_facets):
```

Comment:
Why is this moved to util? Is this method used somewhere else?

Reply:
Yeah, in `export_url_controller`.